src/ntfs/mod.rs

```diff
@@ -507,7 +508,7 @@ mod tests {
                     ("feed_creation_date".to_string(), "20190403".to_string()),
                     (
                         "feed_creation_datetime".to_string(),
-                        "2019-04-03T17:19:00+00:00".to_string()
+                        "2019-04-03T17:19:00Z".to_string()
```

Comment:
The problem on this line is about writing, not reading. The `time` library does write `Z` when the offset is `+00:00` instead of writing the whole `+00:00`. You can see [here](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=df5720c253311846ff1ecc527c511ef4) that `Z` or `+00:00` doesn't make a difference for `time` in parsing/reading (and actually, both are equivalent for ISO-8601).

But it does create a question for other consumers of this NTFS for which we do change the format.

First, it does respect both [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601?oldformat=true#Coordinated_Universal_Time_(UTC)) and the [NTFS specification](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md#format-des-donn%C3%A9es)

But also, I'd be very surprised that consumers do not use a library to parse these dates **and** that the library doesn't support both `+00:00` and `Z`.

So I'd say that the risk of creating issues with this change is very low (except if I missed something).